src/test/java/com/puppycrawl/tools/checkstyle/grammar/comments/AllBlockCommentsTest.java

```diff
@@ -53,8 +52,7 @@ public void testAllBlockComments() throws Exception {
                 createModuleConfig(BlockCommentListenerCheck.class);
         final String path = getPath("InputFullOfBlockComments.java");
         lineSeparator = CheckUtil.getLineSeparatorForFile(path, StandardCharsets.UTF_8);
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, path, expected);
+        execute(checkConfig, path);
```

Comment:
Conceptually this update is right for what we requested in issue.

But there is still problem with ease of test understanding. As author of the PR is advanced in our code base. I decided to request a bit more.

What we try to archive but such test is fact that all block comments a noticed by Check, and it put violation on each of them.
Please put update module to reuse embedded config. It will force us to remove ALL_COMMENTS , and put violations with message of block comment content.
We need to extend Input file to have single line comments with violations.
In PR where I helped you to introduce definition of multiple violation for same line, please reuse this technique.